packages/components/src/components/hds/form/field/index.ts

```diff
@@ -26,6 +26,7 @@ export const LAYOUT_TYPES = Object.values(HdsFormFieldLayoutValues);
 
 export interface HdsFormFieldSignature {
   Args: {
+    id?: string;
```

Comment:
lol, we missed one :)

Reply:
yes, because it's only passed in when called as a contextual component

Reply:
Ok, so we did right in the other. Good :)